docs/resources/aws_ec2_instance.md

```diff
@@ -54,4 +54,46 @@ The following examples show how to use this InSpec audit resource.
 
 ## Matchers
 
-For a full list of available matchers (such as `be_running`) please visit our [matchers page](https://www.inspec.io/docs/reference/matchers/).
+This InSpec audit resource has the following special matchers. For a full list of available matchers (such as `exist`) please visit our [matchers page](https://www.inspec.io/docs/reference/matchers/).
+
+### be_pending
+
+The `pending` matcher tests if the described EC2 instance state is `pending`
+
+    it { should be_pending }
+
+### be_running
+
+The `be_running` matcher tests if the described EC2 instance state is `running`
+
+    it { should be_running }
+
+### be_shutting_down_
+
+The `be_shutting_down` matcher tests if the described EC2 instance state is `shutting-down`
```

Comment:
I'm wondering - some states are valid, but very transitory.  shutting_down, pending, stopping - those might just be mentioned with less prose, as they are much less likely to be of use.

Reply:
Indeed, I was wondering about that when I was documenting them. Only thing I can think is if someone wanted to write tests for something but didn't want to wait for that action to finish.

E.g. a script that spins up nodes and then runs InSpec to verify all are `pending`...it's a stretch.

Would you like me to remove the transitory ones?

Reply:
I think it's fine as-is.  Sorry, not great collaboration to think aloud on a review :/

Reply:
I disagree. We're human and I rather think in the open than in public. Especially in this case. I guarantee anyone else looking at this doc would have thought the same thing.